addons/cookoff/XEH_postInit.sqf

```diff
@@ -27,7 +27,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
     };
 
     _vehicle addEventHandler ["HandleDamage", {
-        if ((_this select 0) getVariable [QGVAR(enable), GVAR(enable)]) then {
+        if ((_this select 0) getVariable [QGVAR(enable), GVAR(enable)] !=0 ) then {
```

Comment:
This will error if someone was using `_vehicle setVariable ["ace_cookoff_enable", true, true]`.

Reply:
Any thoughts on how to mitigate this? I don't know what the general thought process is on people directly setting variables rather than using CBA settings or Eden check boxes

Reply:
Could use in: `(_this select 0) getVariable [QGVAR(enable), GVAR(enable)] in [1, 2, true]`.